server/aws-lsp-codewhisperer/src/language-server/chat/telemetry/chatTelemetryController.ts

```diff
@@ -85,8 +88,13 @@ export class ChatTelemetryController {
         return tabId && this.#tabTelemetryInfoByTabId[tabId]?.conversationId
     }
 
+    public getCustomizationId(tabId?: string, messageId?: string) {
```

Comment:
nit: I think params should not be optional, since function does nothing otherwise.

Reply:
spotted this too, +1